modules/table/src/main/java/org/apache/ignite/internal/table/distributed/replicator/PartitionReplicaListener.java

```diff
@@ -2382,18 +2422,18 @@ private CompletableFuture<Boolean> resolveTxState(
             UUID txId,
             HybridTimestamp timestamp
     ) {
-        requireNonNull(timestamp, "timestamp");
+        boolean readLatest = timestamp == null;
 
         return placementDriver.sendMetaRequest(commitGrpId, FACTORY.txStateReplicaRequest()
                         .groupId(commitGrpId)
-                        .readTimestampLong(timestamp.longValue())
+                        .readTimestampLong((readLatest ? HybridTimestamp.MIN_VALUE : timestamp).longValue())
```

Comment:
It seems incorrect. Why we propagate MIN_VALUE as a read timestamp?

Reply:
I am sure it is correct, because I tested it. 
The timestamp is set MIN_VALUE , because it gives a guarantee that this request don't wait for safe time.

Reply:
> I am sure it is correct, because I tested it.
What exactly did you test? Where are the tests?

I mean that RW transaction has both startTimestamp and corresponding operations timestamps. Why we use MIN_VALUE for RW instead of actual RW related timestamps?

> because it gives a guarantee that this request don't wait for safe time.
Could you please share a code, where we wait for a safeTime for the writeIntent resolution?

Reply:
Maybe nowhere, but we also use this timestamp to update a clock on the other node. The minimum value protects us from a useless update.
Moreover, I do not see the need to propagate a specific timestamp for RW.

Reply:
Discussed in person. Let' it be MIN_VALUE.